### PR TITLE
Releases angelfish - how good

### PR DESCRIPTION
**Pull request by andycarrell**
LFG

Also, pretty sure we can remove tailwind next 🕺

### CODE REVIEW

File: app/pages/angelfish.tsx [hunk 1]
```diff
@@ -110,9 +47,8 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
 
 export default Home;
 
-export const getServerSideProps: GetServerSideProps<HomePageProps> =
-  async () => {
-    return {
-      props: await fetchHomePageProps(),
-    };
+export const getServerSideProps = (): { notFound: boolean } => {
```
Comment (jishaal) [inline app/pages/angelfish.tsx:50]: Oh nice so we keeping this for vNext?

File: app/pages/index.tsx [hunk 2]
```diff
@@ -1,129 +1,92 @@
...
-          <div className="flex flex-row justify-content-between">
-            <div>
-              <h2 className="text-2xl font-light text-[color:var(--text-primary)]">
-                Vaccination rates
-              </h2>
-              <h1 className="text-6xl font-bold leading-[4.5rem] text-[color:var(--text-primary)] mb-6 md:mb-8">
-                Auckland
-              </h1>
+    <main className="page flex align-items-center">
+      <Head>
```
Comment (jishaal) [inline app/pages/index.tsx:31]: We need to remove this `<Head>` and children too

**Review by jishaal — CHANGES_REQUESTED** [review]

File: app/pages/angelfish.tsx (hunk 1 above)
Comment (andycarrell) [reply app/pages/angelfish.tsx:50]: That's the plan ;)

File: app/pages/index.tsx (hunk 2 above)
Comment (andycarrell) [reply app/pages/index.tsx:31]: Ofc will wait for your PR too

Comment (jishaal) [reply app/pages/index.tsx:31]: Should we merge that into this and have single release PR

Comment (andycarrell) [reply app/pages/index.tsx:31]: Na I think maybe even close this, there's two PRs now 😬

Comment (jishaal) [reply app/pages/index.tsx:31]: I don't get it 😅 Can always merge around the same time

**Comment by finnclark** [comment]
OOOhhhhhHHHhhHhH baby bring this bad boi up to date and then 🚀

**Review by finnclark — APPROVED** [review]
lfg

**Review by jishaal — APPROVED** [review]

**Pull request merged**